[libxml2] Failed to build

```
make[2]: Entering directory '/usr/local/tmp/crew/libxml2-v2.9.10.tar.bz2.dir/libxml2-v2.9.10'
  CC       xmllint.o
  CC       SAX.lo
  CC       entities.lo
  CC       encoding.lo
encoding.c: In function ‘xmlEncOutputChunk’:
encoding.c:1961:31: error: ‘TRUE’ undeclared (first use in this function)
 1961 |                               TRUE);
      |                               ^~~~
encoding.c:1961:31: note: each undeclared identifier is reported only once for each function it appears in
make[2]: *** [Makefile:1287: encoding.lo] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: Leaving directory '/usr/local/tmp/crew/libxml2-v2.9.10.tar.bz2.dir/libxml2-v2.9.10'
make[1]: *** [Makefile:1479: all-recursive] Error 1
make[1]: Leaving directory '/usr/local/tmp/crew/libxml2-v2.9.10.tar.bz2.dir/libxml2-v2.9.10'
make: *** [Makefile:893: all] Error 2
libxml2 failed to install: `make V=0 -j2` exited with 2
```

--- a/packages/libxml2.rb
+++ b/packages/libxml2.rb
@@ -3,30 +3,34 @@ require 'package'
 class Libxml2 < Package
   description 'Libxml2 is the XML C parser and toolkit developed for the Gnome project.'
   homepage 'http://xmlsoft.org/'
-  version '2.9.10'
+  version '2.9.10-1'
   compatibility 'all'
   source_url 'https://gitlab.gnome.org/GNOME/libxml2/-/archive/v2.9.10/libxml2-v2.9.10.tar.bz2'
   source_sha256 '5f1cc19c849cccabb983881cf1ebf833f42db5d7b8afba7a7763a2ac3101715c'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libxml2-2.9.10-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libxml2-2.9.10-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libxml2-2.9.10-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libxml2-2.9.10-chromeos-x86_64.tar.xz',
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libxml2-2.9.10-1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libxml2-2.9.10-1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libxml2-2.9.10-1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libxml2-2.9.10-1-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: '6c7da2f6074adc4022ade9e72a070037657a1c708150404c5b9c7d07634c5390',
-     armv7l: '6c7da2f6074adc4022ade9e72a070037657a1c708150404c5b9c7d07634c5390',
-       i686: '9baf9a41ad01e2b6d09f09c9a65a74b37ac839e53ce00c0ebf048bc1d1d7b01f',
-     x86_64: '4c1305434bbac822e6d08234d3058e94bca0ab95591ca25a0ccc8675cd16c48e',
+    aarch64: '5ee3b6147dfce492b631c53893d1cc06e809e204498241710296035e49aa9d57',
+     armv7l: '5ee3b6147dfce492b631c53893d1cc06e809e204498241710296035e49aa9d57',
+       i686: 'd498f4cf27a067bf3d8257f67c6e1a405908460246a15af995991ec1f3c64c74',
+     x86_64: '2082401a85b8e61263556f9fa0612d1d74064db0145a2fac0756c121bc0f11c4',
   })
 
   depends_on 'zlibpkg'
 
+  def self.patch
+    # Fix encoding.c:1961:31: error: ‘TRUE’ undeclared (first use in this function)
+    system "for f in \$(grep -rl \'TRUE)\'); do sed -i 's,TRUE),true),g' \$f; done"
+  end
+
   def self.build
-    system './autogen.sh'
     # libxml2-python built in another package (libxml2_python)
-    system "./configure #{CREW_OPTIONS} --enable-shared --disable-static --with-pic \
+    system "./autogen.sh #{CREW_OPTIONS} --enable-shared --disable-static --with-pic \
 --without-python --without-lzma --with-zlib --with-icu --with-threads --with-history"
     system 'make'
   end

